display_list/skia/dl_sk_canvas.cc

```diff
@@ -325,6 +325,14 @@ void DlSkCanvasAdapter::DrawTextBlob(const sk_sp<SkTextBlob>& blob,
   delegate_->drawTextBlob(blob, x, y, ToSk(paint));
 }
 
+void DlSkCanvasAdapter::DrawTextFrame(
+    const std::shared_ptr<impeller::TextFrame>& text_frame,
+    SkScalar x,
+    SkScalar y,
+    const DlPaint& paint) {
+  FML_CHECK(false);
```

Comment:
I could essentially remove this check, that would prevent apps that run with --enable-impeller but fallback to software from crashing - though text wont render correctly.

Reply:
This will be fixed in https://github.com/flutter/engine/pull/46124 so I will leave the CHECK here